modules/cms-kit/test/Volo.CmsKit.Application.Tests/Blogs/BlogPostPublicAppService_Tests.cs

```diff
@@ -62,4 +73,49 @@ public async Task GetAsync_ShouldThrowException_WithNonExistingBlogSlug()
 
         exception.EntityType.ShouldBe(typeof(Blog));
     }
+
+    [Fact]
+    public async Task GetListAsync_ShouldFilterByUser()
+    {
+        var user2 = await userRepository.GetAsync(cmsKitTestData.User2Id);
+        var blog = await blogRepository.GetAsync(cmsKitTestData.Blog_Id);
+        
+        var blogPost = await blogPostManager.CreateAsync(
+            user2,
+            blog,
+            cmsKitTestData.BlogPost_1_Title + "by user2",
+            cmsKitTestData.BlogPost_1_Slug + "by user2",
+            "Short desc 1",
+            "Blog Post 1 Content"
+        );
+
+        await blogPostRepository.InsertAsync(blogPost);
+
+        //should get all not filtered by user
+        var blogPosts = await blogPostAppService.GetListAsync(cmsKitTestData.BlogSlug,
```

Comment:
We can use pre-seeded data from here (https://github.com/abpframework/abp/blob/dev/modules/cms-kit/test/Volo.CmsKit.TestBase/CmsKitDataSeedContributor.cs#L354-L382)

Or we can move this adding new data logic to **CmsKitDataSeedContributor**. Otherwise this test case will test the insert case too

Reply:
If i move it to CmsKitDataSeedContributor it will affect all other tests. Is it a good way to change all the tests when we just want to check simple new added filter? It ends up replacing all existing relevant tests for each new feature we add.

Reply:
For example I added some tests here https://github.com/abpframework/abp/pull/12073. I updated 3 test classes here. They creates new blogpost 13 times. Should I also use CmsKitDataSeedContributor here. Because I will have to change all other test when I add every single test case.

Reply:
ok, got it. It's ok to keep here for this case 👍 :)